### DISCUSSION TITLE
Multi booking form

### DISCUSSION THREAD

**Discussion by cyprienluyet** [liedekef/events-made-easy#242] [Q&A]
Hello everyone,
I've been trying for a few days to make a reservation form for a mountain hut open in summer.
I've created a recurring event with the hut's opening dates, the different prices categories and the number of places available per day. So far, so good.
My problems start when people want to book several nights in the refuge. If I make a special field with the desired number of days, the places available in the next day's event are not counted. I'm trying to create a multiple event, but I can't create the necessary conditions. Ideally, I'd like to choose the number of days, then the date of the first night and, if the number of days isn't equal to one, have the reservation updated in the following events.
Can anyone help me?

Translated with www.DeepL.com/Translator (free version)

**Comment by liedekef** [comment]
A booking for one event does not influence other events. For your multibooking form, the only way is to have people indicate all days (read: 1 event per day).
EME is currently not designed to provide a camping-like reservation setup ...

**Comment by cyprienluyet** [comment]
Thanks for your fast answer

**Comment by liedekef** [comment]
But I think I might have a solution in mind ... if I code up a "max spaces per location" setting, then I could check all events using that location and count the number of seats taken at the same time, so I could prevent to surpass the number set in the location.
That would immediately open up new possibilities :-)

**Reply by liedekef** [reply]
This will be in the next version btw, so you can have a multibooking form with several events taking up the same location and if the location has a limit, that limit will be enforced over all bookings for events happening at the same time at that location.

**Comment by cyprienluyet** [comment]
J'y Franky,

Thanks for your job.
I ll try it.
Have a Nice weekend
